packages/pros-devices/README.md

```diff
@@ -0,0 +1,17 @@
+# pros-devices
+
+functionality for accessing hardware connected to the V5 brain.
+
+# Overview
+
+The V5 brain features 21 RJ9 4p4c connector ports (known as "Smart ts")for communicating with
+newer V5 peripherals, as well as six 3-wire ports with log-to-digitalconversion capability for
+compatibility with legacy cortex devices. This module provides access both smart devices and
+ADI devices.
```

Comment:
Might be good to just look over this section again because I'm noticing some missing spaces, incorrect capitalization, and typos.

Reply:
This is copied from the module docs so that's another place that should be fixed. The purpose of this PR is just getting a new version uploaded and I'm willing to fix this later but I think it's out of scope.